src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -336,7 +341,8 @@ private CommandResult executeCommand(String commandText)
                         .select(switchTabCommandResult.getTabToSwitchIndex());
                 if (switchTabCommandResult.getTabToSwitchIndex() == STATS_TAB_INDEX) {
                     ObservableList<DayData> customQueue = logic.getCustomQueue();
-                    statisticsDisplay.updateGraphs(customQueue);
+                    statisticsManager.updateStatistics(customQueue);
+                    this.updateStatisticsDisplay();
```

Comment:
we can't call this within statisticsDisplay itself? 
[just thinking of ways to reduce the crowding in MainWindow]

Reply:
not with current implementation as mainwindow ties manager to display!